main.py

```diff
@@ -8,7 +8,7 @@
 
 from bot_globals import client, logger
 from database import init_mongodb_conn
-from utils.message_scheduler import send_daily_question_and_update_stats
+from utils.message_scheduler import send_daily_question_and_update_stats, send_daily_question_and_update_stats_schedule
```

Comment:
Oh wow, I'm surprised no errors were thrown when I wrote the code for this yesterday. No wonder the stats weren't updated last night and the notifications weren't sent (only just saw that rn)